internal/app/wwctl/container/exec/child/main.go

```diff
@@ -165,11 +172,8 @@ func CobraRunE(cmd *cobra.Command, args []string) (err error) {
 	os.Setenv("PATH", "/bin:/sbin:/usr/bin:/usr/sbin:/usr/local/bin:/usr/local/sbin")
 	os.Setenv("HISTFILE", "/dev/null")
 
-	_ = syscall.Exec(args[1], args[1:], os.Environ())
-	/*
-		Exec replaces the actual program, so nothing to do here afterwards
-	*/
-	return nil
+	wwlog.Debug("Exec: %s %s", args[1], args[1:])
+	return syscall.Exec(args[1], args[1:], os.Environ())
```

Comment:
@mslacken I remembered you made this change in the dracut branch but it didn't get merged there; so I brought it in here.